recipe/meta.yaml

```diff
@@ -11,8 +11,10 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
   noarch: python
+  script:
+    - python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
I clearly did not understand the impact of this line!

I wonder if cutting a new release of PySurfer would allow us to get rid of this, but in the meantime might as well restore it. Added a test that now passes that shows `import surfer` passes (as does a check of the version)